recipes/pyjion/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set name = "pyjion" %}
+{% set version = "1.2.6" %}
```

Comment:
![image](https://user-images.githubusercontent.com/881019/152985280-da753f0d-0efc-4ca4-bdae-93dbca79278f.png)

The commit hash (https://github.com/tonybaloney/Pyjion/commit/66a772d48825e2511b9c6e8bbcc739d84f84e800) could be used instead if there are concerns the tag could change?